external/test-runtime/XUnit.Runtime.depproj

```diff
@@ -140,7 +140,10 @@
           AfterTargets="RestorePackages"
           Condition="'$(TargetGroup)' == 'uapaot'">
     <PropertyGroup>
-      <TestILCToolsPackageName>TestILCNugetPackageForCoreFX</TestILCToolsPackageName>
+      <TestILCToolsPackageName>TestILC</TestILCToolsPackageName>
+      <TestILCToolsPackageName Condition="'$(ArchGroup)' == 'x64'">$(TestILCToolsPackageName).amd64ret</TestILCToolsPackageName>
+      <TestILCToolsPackageName Condition="'$(ArchGroup)' == 'x86'">$(TestILCToolsPackageName).x86ret</TestILCToolsPackageName>
+      <TestILCToolsPackageName Condition="'$(ArchGroup)' == 'arm'">$(TestILCToolsPackageName).armret</TestILCToolsPackageName>
```

Comment:
Should there be some special handling if none of the conditions are met? I'm wondering if it might be hard to debug if this property ends up as just `TestILC`.